apps/examples/src/HomePage.tsx

```diff
@@ -14,22 +14,11 @@ export function HomePage() {
 				</p>
 			</div>
 			<ul className="examples__list">
-				{examples
-					.filter((example) => !example.hide)
```

Comment:
this logic doesn't apply anymore?

Reply:
There are one or two hidden examples, not sure why they've been hidden though.

Reply:
Oh, ok - then we should keep this logic around?